test/acceptance/partner_profile.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-env mocha */
+import { setup, teardown } from './helpers'
+import { server as antigravity } from 'antigravity'
+
+describe('Partner profile page', () => {
+  let gravity, browser
+
+  before(async () => {
+    ({ gravity, browser } = await setup())
+    gravity.use(antigravity)
```

Comment:
Nope, not required—and I'd encourage not using it in future tests to leave more control to the individual test suites.